t/TestUtil.pm

```diff
@@ -0,0 +1,150 @@
+package TestUtil;
+
+use strict;
+use warnings;
+
+use Test::More;
+
+use Zonemaster::Engine;
+use Zonemaster::Backend::Config;
+use Zonemaster::Backend::RPCAPI;
+use Zonemaster::Backend::TestAgent;
```

Comment:
> One drawback to loading both the RPCAPI and the TestAgent module in all(?) our test files is that the unit tests might not catch missing use statments at test-time that then could surface at run-time.

I'm not sure I have a strong enough knowledge on namespace and scopes to fully get the underlying issue here.

> Maybe the TestUtil module could take arguments so that the caller can control which part is loaded? E.g. something like this to actually load both:

Using the Exporter module? Then I'm not sure how we could prevent loading one part or another. Would moving the `use Zonemaster::Backend::RPACPI` (resp `TestAgent`) within the `create_rpacpi()` method (resp `create_testagent()`) gives the expected behavior?

Reply:
Thanks for the detailed example!

Reply:
No problem! I needed to convince myself anyway that I didn't send you down a wild goose chase :)